src/ibc_api/utils.py

```diff
@@ -52,7 +52,7 @@ def _authenticate(token_dir=TOKEN_ROOT):
             # set the token
             siibra.set_ebrains_token(token)
```

Comment:
```suggestion
            EbrainsRequest.set_token(token)
```
We should probably fix it for subsequent runs, too.